casadm/cas_lib.c

```diff
@@ -1919,6 +1946,43 @@ int remove_core(unsigned int cache_id, unsigned int core_id,
 	return SUCCESS;
 }
 
+int remove_inactive_core(unsigned int cache_id, unsigned int core_id)
+{
+	int fd = 0;
+	struct kcas_remove_inactive cmd;
+
+	/* don't even attempt ioctl if filesystem is mounted */
+	if (SUCCESS != check_if_mounted(cache_id, core_id)) {
+		return FAILURE;
```

Comment:
please add some error message here

Reply:
Error message is inside `check_if_mounted`.